bylaws/3-cf-council.md

```diff
@@ -45,75 +44,74 @@ An up-to-date list of CFC members will be maintained in the `cf-council` [CONTAC
 
-- **Officers and Staff:** The CFC may appoint CF general members as officers or hire staff to manage specific administrative functions. These officers and staff are not considered CFC members and do not have voting rights. 
-  - Officers may be invited to attend CFC meetings as non-voting participants
-  - Officers may resign at any time by notice to the CFC.
+#### CFC Chairperson
 
-### CFC Chairperson
+The Chairperson is instrumental in steering the operations of the CFC. Their key responsibilities are to:
 
-The Chairperson orchestrates regular CFC meetings, ensuring leadership in discussions is evenly spread. They also oversee the proper handling of [Code of Conduct reports and escalations][coc-reports].
+- Organize and moderate regular CFC leadership meetings, ensuring balanced participation and effective discussion.
+- Facilitate the progression of votes, ensuring that decisions are efficiently managed and accurately recorded.
+- Oversee the processing of [Code of Conduct reports and escalations][coc-reports], upholding the CF's ethical standards and integrity.
 
-#### Officers and Staff
+This role is pivotal for the orderly and fair functioning of the CFC. While crucial in guiding processes and meetings, the Chairperson does not possess additional decision-making power or authority in setting the CFC's strategic direction. Their focus is on ensuring that the Council's operations are conducted effectively and impartially.
 
-The CFC may appoint CF general members as officers or hire staff to manage specific administrative functions. These officers and staff are not considered CFC members and do not have voting rights.
+## Extended Governance Committee (EGC)
 
-- Officers may be invited to attend CFC meetings as non-voting participants
-- Officers may resign at any time by notice to the CFC.
+Expanding beyond the CFC, the Extended Governance Committee (EGC) plays a vital role in encompassing broader representation and decision-making within the CF.
+
+- **Composition:** The EGC includes every member of the CF Council and one nominated leader from each CF project.
+- **Role:** The EGC is responsible for making key decisions that have a [significant impact on the CF and its projects](#voting).
+
+This structure ensures a balanced representation, combining strategic oversight with project-specific insights.
+
+An up-to-date list of Project Leads as members of the EGC will be maintained in the `egc` [CONTACTS.yaml][] attribute.
 
 ## Meetings, Procedures and Decision-Making
 
-The CFC conducts meetings primarily through electronic means (like Zoom or Google Meet) to accommodate its globally distributed and remote members. Key points:
+The CFC uses electronic and asynchronous methods for meetings and decision-making, catering to our globally distributed membership. This facilitates inclusive participation and effective governance.
```

Comment:
Asynchronous meetings are a fine idea, but I'm not sure how it translates in practice :grin: 

```suggestion
The CFC uses electronic and (where possible) asynchronous methods for meetings and decision-making, catering to our globally distributed membership. This facilitates inclusive participation and effective governance.
```